Cannot build documentation without successful "import numpy" error when having built numpy to a virtual enviornment

## Documentation

Issue in file `doc/source/docs/howto_build_docs.rst`
Upon building numpy in a virtual environment and then trying to build documentation, `make html` and other document building commands throw an error saying:
```
numpy not found, cannot build documentation without successful "import numpy"
make: *** [Makefile:90: version-check] Error 1
```
This happens because `Makefile` defines `PYTHON=python$(PYVER)` which when executed later, runs  `make html` command using the global python installation instead of using the virtual environment (yes, even when the current shell session has the virtual environment activated).  It could be documented in the `howto_build_docs.rst` to override `Makefile`'s `PYTHON` at the command line while running `make html` by simply changing it to `make PYTHON=python html`.

--- a/doc/source/docs/howto_build_docs.rst
+++ b/doc/source/docs/howto_build_docs.rst
@@ -56,7 +56,11 @@ in the ``doc/`` directory. If all goes well, this will generate a
 a message about ``installed numpy != current repo git version``, you must
 either override the check by setting ``GITVER`` or re-install NumPy.
 
-Note that if you have built numpy into a virtual enviornment and getting an error that says ``numpy not found, cannot build documentation without...`` you need to override the makefile ``PYTHON`` variable at the commandline, so instead of writing ``make html`` write::
+Note that if you have built numpy into a virtual enviornment and 
+getting an error that says ``numpy not found, cannot build 
+documentation without...`` you need to override the makefile 
+``PYTHON`` variable at the commandline, so instead of writing ``make 
+html`` write::
 
     make PYTHON=python html
     

--- a/doc/source/docs/howto_build_docs.rst
+++ b/doc/source/docs/howto_build_docs.rst
@@ -56,13 +56,10 @@ in the ``doc/`` directory. If all goes well, this will generate a
 a message about ``installed numpy != current repo git version``, you must
 either override the check by setting ``GITVER`` or re-install NumPy.
 
-Note that if you have built numpy into a virtual enviornment and getting
-an error that says ``numpy not found, cannot build documentation without successful "import numpy"``
-you need to override the makefile variable at the commandline so instead of writing ``make html``
-write::
+Note that if you have built numpy into a virtual enviornment and getting an error that says ``numpy not found, cannot build documentation without...`` you need to override the makefile variable at the commandline, so instead of writing ``make html`` write::
 
     make PYTHON=python html
-
+    
 
 Note that building the documentation on Windows is currently not actively
 supported, though it should be possible. (See Sphinx_ documentation

--- a/doc/source/docs/howto_build_docs.rst
+++ b/doc/source/docs/howto_build_docs.rst
@@ -48,12 +48,21 @@ with::
 
 Now you are ready to generate the docs, so write::
 
+    cd doc
     make html
 
 in the ``doc/`` directory. If all goes well, this will generate a
 ``build/html`` subdirectory containing the built documentation. If you get
 a message about ``installed numpy != current repo git version``, you must
 either override the check by setting ``GITVER`` or re-install NumPy.
+
+Note that if you have built numpy into a virtual enviornment and getting
+an error that says ``numpy not found, cannot build documentation without successful "import numpy"``
+you need to override the makefile variable at the commandline so instead of writing ``make html``
+write::
+
+    make PYTHON=python html
+
 
 Note that building the documentation on Windows is currently not actively
 supported, though it should be possible. (See Sphinx_ documentation

--- a/doc/source/docs/howto_build_docs.rst
+++ b/doc/source/docs/howto_build_docs.rst
@@ -56,7 +56,7 @@ in the ``doc/`` directory. If all goes well, this will generate a
 a message about ``installed numpy != current repo git version``, you must
 either override the check by setting ``GITVER`` or re-install NumPy.
 
-Note that if you have built numpy into a virtual enviornment and getting an error that says ``numpy not found, cannot build documentation without...`` you need to override the makefile variable at the commandline, so instead of writing ``make html`` write::
+Note that if you have built numpy into a virtual enviornment and getting an error that says ``numpy not found, cannot build documentation without...`` you need to override the makefile ``PYTHON`` variable at the commandline, so instead of writing ``make html`` write::
 
     make PYTHON=python html
     

--- a/doc/source/docs/howto_build_docs.rst
+++ b/doc/source/docs/howto_build_docs.rst
@@ -56,11 +56,10 @@ in the ``doc/`` directory. If all goes well, this will generate a
 a message about ``installed numpy != current repo git version``, you must
 either override the check by setting ``GITVER`` or re-install NumPy.
 
-Note that if you have built numpy into a virtual enviornment and 
-getting an error that says ``numpy not found, cannot build 
-documentation without...`` you need to override the makefile 
-``PYTHON`` variable at the commandline, so instead of writing ``make 
-html`` write::
+If you have built numpy into a virtual environment and get an error
+that says ``numpy not found, cannot build documentation without...``,
+you need to override the makefile ``PYTHON`` variable at the command
+line, so instead of writing ``make  html`` write::
 
     make PYTHON=python html
     

